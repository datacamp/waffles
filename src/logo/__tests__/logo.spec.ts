import * as logos from '../index';

describe('Logo', () => {
  it('exports every logo', () => {
    expect(logos).toMatchSnapshot();
  });
});
