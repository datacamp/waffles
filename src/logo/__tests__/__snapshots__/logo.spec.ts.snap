// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo exports every logo 1`] = `
Object {
  "ExcelLogo": [Function],
  "GitLogo": [Function],
  "OracleLogo": [Function],
  "PowerBiLogo": [Function],
  "PythonLogo": [Function],
  "RLogo": [Function],
  "ScalaLogo": [Function],
  "ShellLogo": [Function],
  "SparkLogo": [Function],
  "SpreadsheetsLogo": [Function],
  "SqlLogo": [Function],
  "TableauLogo": [Function],
}
`;
